vendor/bat-native-ledger/src/bat/ledger/internal/uphold/uphold.cc

```diff
@@ -76,12 +66,29 @@ void Uphold::StartContribution(const std::string &viewing_id,
   }
 }
 
-void Uphold::ContributionCompleted(ledger::Result result,
-                                   bool created,
-                                   const std::string &viewing_id) {
+void Uphold::ContributionCompleted(
+    ledger::Result result,
+    bool created,
+    const std::string& viewing_id,
+    const double fee,
+    const ledger::ExternalWallet& wallet) {
   const auto reconcile = ledger_->GetReconcileById(viewing_id);
   const auto amount = ConvertToProbi(std::to_string(reconcile.fee_));
 
+  if (result == ledger::Result::LEDGER_OK) {
```

Comment:
we needed to move 5% fee transaction after publisher transaction was successful. Before both were triggered at the same time